src/status_im2/contexts/quo_preview/main.cljs

```diff
@@ -318,7 +319,11 @@
                        {:name      :reorder-item
                         :component reorder-item/preview-reorder-item}
                        {:name      :category
-                        :component category/preview}]
+                        :options   {:topBar {:visible true}}
+                        :component category/preview}
+                       {:name      :data-item
+                        :options   {:topBar {:visible true}}
+                        :component data-item/preview-data-item}]
```

Comment:
The options can be removed. The top bar is added by default to quo2 preview.

Reply:
Done